### PR TITLE
Support for Rails 5.0 to 6.1

### PR DESCRIPTION
**Pull request by mathieujobin**
- run tests on Travis
- only a couple of edge case I have not solved
- rebuilt the gem structure to be freshed
- rename to push to a fresh
- currently only 5.x is supported
- I got branches for 4.x if anyone asks

### CODE REVIEW

**Comment by mathieujobin** [comment]
Note I have a 5.0 app running on `bb3a47f` equivalent for more than a year
Although I had flash disabled and I could not figure out how to make it work with 5.1
solved now

File: lib/rails/render/component/version.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Rails
+  module Render
+    module Component
+      VERSION = "5.0.1"
```
Comment (mathieujobin) [inline lib/rails/render/component/version.rb:6]: Previous version we're following rails... I wonder if it's adequate to keep the same logic, if I am going to support all 5.x and 6.x ? Any opinion?

File: lib/render_component/components.rb [hunk 2]
```diff
@@ -121,8 +129,8 @@ def request_for_component(controller_path, options)
             request_env['REQUEST_URI'] = url_for(options)
             request_env["PATH_INFO"] = url_for(options.merge(:only_path => true))
             request_env["action_dispatch.request.symbolized_path_parameters"] = request_params
-            request_env["action_dispatch.request.parameters"] = request_params.with_indifferent_access
-            request_env["action_dispatch.request.path_parameters"] = Hash[request_params.select{|key, value| [:controller, :action].include?(key)}].with_indifferent_access
+            request_env["action_dispatch.request.parameters"] = request_params.symbolize_keys
+            request_env["action_dispatch.request.path_parameters"] = Hash[request_params.select{|key, value| [:controller, :action].include?(key)}].symbolize_keys
```
Comment (mathieujobin) [inline lib/render_component/components.rb:133]: Not my change, I don't know if maybe we should have kept `with_indifferent_access`

**Comment by mathieujobin** [comment]
released as https://rubygems.org/gems/rails-render-component
